bindings/pydeck/tests/io/test_html.py

```diff
@@ -1,8 +1,6 @@
 import html
 import webbrowser
 
-import IPython
```

Comment:
We still always have access to `IPython` in tests, right?

Reply:
That's correct, it's still available through the dev requirements installed during testing.